apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-textbox.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, ChangeEvent, useEffect, useMemo } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { gql, useLazyQuery } from '@apollo/client';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './location-autocomplete-textbox.scss';
+import { faMapMarkerAlt, faLocationArrow } from '@fortawesome/free-solid-svg-icons';
+import { debounce } from 'lodash';
+
+export const AUTOCOMPLETED_LOCATIONS = gql`
+  query Query($userInput: String!) {
+    getAutocompletedLocations(userInput: $userInput)
+  }
+`;
+
+export interface LocationAutocompleteTextboxProps {
+  handleClose: (value: string) => void;
+  input: string;
+}
+
+export default function LocationAutocompleteTextbox({ handleSetValue, input }) {
+  const [open, setOpen] = useState(false);
+  const [options, setOptions] = useState<string[]>([]);
+  const [getAutocompletedLocations, { loading, data }] = useLazyQuery(AUTOCOMPLETED_LOCATIONS);
+  const CURRENT_LOCATION = 'Current Location';
+  const debounceGetLocation = useMemo(
+    () =>
+      debounce(
+        (values: string) =>
+          getAutocompletedLocations({
+            variables: { userInput: values },
+          }),
+        350
+      ),
+    []
+  );
+
+  useEffect(() => {
+    if (!loading && data) {
+      setOptions(data.getAutocompletedLocations);
+    }
+  }, [loading, data, setOptions]);
+
+  const getCurrentPosition = () => {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition((position) => {
+        let pos = `latitude: ${position.coords.latitude}, longitude: ${position.coords.longitude}`;
+        handleSetValue({ title: 'Current Location', coords: pos });
+      });
+    }
+  };
+
+  return (
+    <Autocomplete
```

Comment:
If it's possible, can we  remove the box shadow and make the background white in the auto complete popups 
![Screenshot 2020-12-31 114625](https://user-images.githubusercontent.com/44760251/103419447-26374780-4b61-11eb-9d24-69d178f74b15.png)

Reply:
sure, I will look into that :)

Reply:
Ok so @ys-lin, @bit172  and I looked into it, and we don't see   a box shadow for this and it appears that the options listed in the autocomplete appear to be white to us.

![image](https://user-images.githubusercontent.com/35787279/104263384-8084b200-5457-11eb-898e-df840caffda7.png)